pyiode/iode_python.pxd

```diff
@@ -25,6 +25,7 @@ cdef extern from "iode.h":
 
     # SCR4 functions
     cdef int    SCR_free(void *ptr)
+    cdef int    SCR_free_tbl(char **tbl)
```

Comment:
Please remove this line. 
Note that below I have added the function `free_tbl(char** tbl)` for C++ compatiblity:
```C
cdef extern from "s_pyiode.c":
       cdef int free_tbl(char** tbl)
```

Please replace all call to `SCR_free_tbl` by `free_tbl` everwhere you used `SCR_free_tbl`.

Reply:
Please replace everywhere you call SCR_free_tbl() by free_tbl()

Reply:
Done